src/Pages/Main/Newitem/Newitem.scss

```
@@ -1,2 +1,106 @@
 @import "../../../Styles/common.scss";
-@import "../../../Styles/reset.scss";
+
+* {
+  outline: none;
+}
+.container {
+  margin: 300px;
+  width: 1120px;
+  height: 7187px;
+}
+img {
+  width: 1080px;
```

```suggestion
  height: 7187px;
}

img {
  width: 1080px;
```
새로운 셀렉터 앞에서는 한줄 띄워주세요 :)

---

넷!!

---

네엡!